### ISSUE TITLE
Unable to Install via `naomi-setup.sh`

### ISSUE THREAD

**Issue by omerasif57** [NaomiProject/Naomi#314]
## Description

Using Command available on website: 
`. <( wget -O - https://installers.projectnaomi.com/naomi-setup.sh )`
I tried to install. It went through fine for commands: ( I chose to install in manual steps )

`sudo apt install git`
`sudo apt install update`
`sudo apt install upgrade`

And at a installer shell script command (I forgot the whole command), terminal got terminated. and now when ever I try to run command:
`. <( wget -O - https://installers.projectnaomi.com/naomi-setup.sh )` 
and Write Y: YES. Terminal get terminated.

## Expected Behavior
Installation should proceed within terminal and log the command.

## Actual Behavior
Installation command quits the terminal without a log. 

## Possible Fix
N/A

## Steps to Reproduce

1. use script command: . `<( wget -O - https://installers.projectnaomi.com/naomi-setup.sh )` to install in a ubuntu terminal
2. Chose to install in manual step ( for sudo authentication each time a command needs to run)

## Context

I was try to install the program from a shell script
## Your Environment

* Ubuntu 20.04.1 LTS
* kernel version: 5.8.0-40-generic
* Python 3.8.5 
* GCC 9.3.0

**Comment by aaronchantrill** [comment]
@omerasif57 thank you for reporting this issue. It might take a couple of days for us to track down the issue and solve it. Is your Ubuntu system 64bit? If so, I would bypass the problem by using git to clone the naomi-dev branch (the main branch right now) and then run the naomi-setup.sh script from there.
```
$ git clone https://github.com/NaomiProject/Naomi.git
$ cd Naomi
$ ./naomi-setup.sh
```
If your system is a 32 bit system, there is a known issue because deepspeech is currently a requirement, but I can walk you through it.

**Comment by omerasif57** [comment]
I have 64-bit system. In fact, I tried to run `/Naomi/naomi-setup.sh` manually in a python3 _virtual_ environment with requirements. (`pip install -r /Naomi/python-requirements.txt`)

`Naomi/naomi-setup.sh` downloaded python 2.7.x in `.naomi/local` and broke during setup. I thought, project required 2.7.x for installation. is it correct? 

Anyways, for now I am pending installation for the weekend. I will give it a try again via git cloning as described above. 

Regards.

**Comment by aaronchantrill** [comment]
The latest version of Naomi requires Python 3.5 or above. Only the "stable" version still requires python 2.7. It is really old (December 2018), and nobody should use it. Hopefully we can put out a new release soon. Always choose "milestone" or "nightly".

The naomi-setup.sh automatically creates a python virtual environment for Naomi, so you shouldn't have to do that.

I kind of remember having the problem of Naomi having an issue and closing the terminal before we could see what the issue was. I think we were able to resolve it by redirecting the output to a log file like this:
```
$ Naomi/naomi-setup.sh |& tee ~/naomi-setup.log
```
and then we could read the error in the log file and fix the root cause. I don't remember what the root issue was, though. If that helps you get an error message, please share.

@AustinCasteel can we work on stabilizing the current version and putting out the 3.0 release? I just tried installing and chose the "stable" version which is still labeled as "recommended" and it did the same thing to me (crapped out and shut down the terminal). When I re-ran and captured to a log file, I got this error message:
```
$ . <( wget -O - https://installers.projectnaomi.com/naomi-setup.sh ) |& tee ~/naomi-setup.log
....
NAOMI SETUP:
Naomi is continuously updated. There are three options to choose from:

'Stable' versions are thoroughly tested official releases of Naomi. Use
the stable version for your production environment if you don't need the
latest enhancements and prefer a robust system

'Milestone' versions are intermediary releases of the next Naomi version,
released about once a month, and they include the new recently added
features and bugfixes. They are a good compromise between the current
stable version and the bleeding-edge and potentially unstable nightly version.

'Nightly' versions are at most 1 or 2 days old and include the latest code.
Use nightly for testing out very recent changes, but be aware some nightly
versions might be unstable. Use in production at your own risk!

Note: 'Nightly' comes with automatic updates by default!

  1) Use the recommended ('Stable')
  2) Monthly releases sound good to me ('Milestone')
  3) I'm a developer or want the cutting edge, put me on 'Nightly'
Choice [1-3]: 1 - Easy Peasy!
....
sudo: ./naomi_apt_requirements.sh: command not found
Notice: Error installing apt packages
```
so it looks like stable is anything but.

**Comment by AustinCasteel** [comment]
@omerasif57 I have bumped Naomi so now we have a stable version 3.0, milestone version 3.1.M0, etc. So this should no longer be an issue. I apologize for the inconvenience. If there are any other questions, comments, or concerns please reach out to us here on github or on our discord and we would be more than happy to assist.

**Comment by omerasif57** [comment]
I just tried this 

`. <( wget -O - https://installers.projectnaomi.com/naomi-setup.sh ) |& tee ~/naomi-setup.log`

**Result:**

> 
> WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
> 
> Reading package lists...
> Building dependency tree...
> Reading state information...
> E: Unable to locate package python-pip
> Notice: Error installing apt packages
> 
> Notice: Naomi Apt Setup Wizard Failed.

As per my knowledge, `python-pip` links to python2.7.x pip module. 
I have pip3 available in the system. 
I have python2.7.x as well but pip (module in python2.7.x) is not installed. 

I will now try installing from `git clone <source>` and from there using `shell script` of active dev branch as per @aaronchantrill advice.

**Comment by aaronchantrill** [comment]
Sorry, I wasn't looking at my email last night and missed it. It looks like the python-pip requirement comes from phonetisaurus. I did some work on that a while back to try and make Phonetisaurus compatible with Python3, so I'm not sure if it's still required or not. What distro are you installing on? Anything Debian based should still have a python-pip package.

If you still get this error, try deleting the "python-pip" line from apt_requirements.txt, then when choosing which version to install hit 'S' (shift-S-unshift, lower case 's' doesn't work) to skip the line instead of making a selection. This will prevent the installer from replacing your edit. Then please let us know if that works.

Thanks,
Aaron

**Comment by aaronchantrill** [comment]
I will download Ubuntu 20.04 and try installing on it tomorrow. My internet access has been too unreliable this weekend.

**Comment by aaronchantrill** [comment]
I installed Ubuntu 20.04 on a virtual machine this morning and got the same error about apt being unable to locate the python-pip package. As near as I could tell, they renamed the python-php package to python-pip-whl. Python 2.7 was not installed when I set up the system, but phonetisaurus ran fine without it, so it seems like the Python2 requirements have been removed.

I will remove the  "python-pip" line from apt_requirements.txt and submit a pull request.
